values.yaml

```diff
@@ -86,6 +86,18 @@ st2:
       # Optional name of the imagePullSecret if your custom packs image is hosted by a private Docker registry behind the auth
       #pullSecret: st2packs-auth
 
+    # Custom packs and virtualenvs can be mounted using NFS.
```

Comment:
Great start! :+1: 

I think we'll need a more user-friendly story in both README.md and then after merging this PR at https://docs.stackstorm.com/install/k8s_ha.html which would instruct how to install/use this as alternative to `st2pack` image.

Reply:
I've added more details to the README, when everything is squared on stackstorm-ha repo I'll take a look into updating the main stackstorm doc.